contents/blog/big-five-framework.md

```diff
@@ -0,0 +1,77 @@
+---
+date: 2021-05-25
+title: Big Five - a simple framework for product market fit
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/100x/100x.png
+---
+
+By: [James Hawkins](https://twitter.com/james406), Co-Founder / CEO @ PostHog
+
+Last Friday, James, Tim and Charles had a board meeting with PostHog's lead investors.
```

Comment:
nit: but odd to talk in 3rd p here as we are using "we" everywhere else – maybe change to "Last Friday, we had a board meeting with PostHog's lead investors."